SECURITY.md

```diff
@@ -11,6 +13,32 @@ The following versions are supported with security updates:
 | 1.0.x   | :x:                |
 | < 0.1.0 | :x:                |
 
-## Reporting a Vulnerability
+## Reporting
+
+To report a security vulnerability, please provide the following information:
+
+1. **PROJECT**
+   - Include the URL of the project repository - Example: <https://github.com/orhun/git-cliff>
```

Comment:
This is fixed for this repo and probably won't change, so maybe remove this step?